lib/framework/menu.dart

```diff
@@ -61,9 +61,10 @@ abstract class Menu {
               icon: item['icon'],
               iconLibrary: Utils.optionalString(item['iconLibrary']),
               selected: item['selected'],
-              fab: Utils.optionalBool(item['fab']) ?? false,
-              fabLocation:
-                  Utils.optionalString(item['fabLocation']) ?? 'center',
+              floating: Utils.optionalBool(item['floating']) ?? false,
+              floatingAlignment:
+                  Utils.optionalString(item['floatingAlignment']) ?? 'center',
```

Comment:
use this pattern if you have a default value:
`floating: Utils.getBool(item['floating'], fallback: widget.controller.floating)`
so you don't have to declare the default value twice.